packages/utils/telemetry-utils/src/index.ts

```diff
@@ -11,3 +11,12 @@ export * from "./logger";
 export * from "./mockLogger";
 export * from "./thresholdCounter";
 export * from "./utils";
+export {
+    MonitoringContext,
+    IConfigProviderBase,
+    sessionStorageConfigProvider,
+    mixinMonitoringContext,
+    IConfigProvider,
+    ConfigTypes,
+    loggerToMonitoringContext,
+} from "./config";
```

Comment:
I'm curious why the explicit export here.

Reply:
just minimize the exported surface area. there are things used in test that i don't want exported from the package itself